2-mandatory/2-water-bottle.js

```diff
@@ -17,20 +17,29 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function () {
+  fillUp () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100;
   },
-  pour: function () {
+  pour () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume != 100) {
+      this.volume += 10;
+    }
```

Comment:
This code will work for this particular max capacity configuration, but how can you make it work when you can fill the bottle with any amount in one go, but you don't want the capacity to exceed the total volume of the container?